.github/workflows/unit-tests.yaml

```
@@ -6,7 +6,6 @@ on:
   pull_request:
     branches:
       - main
-      - v0.5
     paths:
       - "**.go"
       - "!e2e/**" # exclude changes in e2e tests
```

This needs to be updated :) 
```suggestion
      - "!test/**" # exclude changes in e2e tests
```